README.md

```diff
@@ -10,7 +9,7 @@ Rippled serialization and transaction signing command-line tool
 ## Table of contents
 
 * [Dependencies](#dependencies)
-  * [ripple-libpp submodule](#ripple-libpp-submodule)
+  * [rippled-libpp submodule](#ripple-libpp-submodule)
```

Comment:
The `-libpp` got left behind in the text. Also rename the anchor to `#rippled-submodule` to match the updated header below.